competition/track2/train/train.py

```diff
@@ -41,19 +70,34 @@ def train(input_path, output_path):
         print("processing scenario " + scenario)
         vehicle_ids = list()
 
+        '''
         for filename in os.listdir(input_path + scenario):
             if filename.endswith(".png"):
                 match = re.search("vehicle-(.*).png", filename)
                 assert match is not None
                 vehicle_id = match.group(1)
                 if vehicle_id not in vehicle_ids:
                     vehicle_ids.append(vehicle_id)
+        '''
+        for filename in os.listdir(input_path + scenario):
+            if filename.endswith(".pkl"):
+                match = re.search("vehicle-(.*).pkl", filename)
+                assert match is not None
+                vehicle_id = match.group(1)
+                if vehicle_id not in vehicle_ids:
+                    vehicle_ids.append(vehicle_id)
 
-        if len(vehicle_ids) < 3:
+        if len(vehicle_ids) < 2:
             continue
         else:
-
-            for id in vehicle_ids[0:2]:
+            if n_vehicles == 'max':
+                n_vehicles = len(vehicle_ids)
+            elif n_vehicles == 'min':
+                n_vehicles = 1
+            elif n_vehicles > len(vehicle_ids):
+                n_vehicles = len(vehicle_ids)
+            
+            for id in vehicle_ids[0:n_vehicles]:
```

Comment:
Semantically the same.
```suggestion
            for id in vehicle_ids[:n_vehicles]:
```